blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GHRepoEx.java

```diff
@@ -13,11 +13,4 @@
  */
 @SuppressFBWarnings(value = "EQ_DOESNT_OVERRIDE_EQUALS", justification = "no extra state added to affect superclass equality")
 public class GHRepoEx extends GHRepository {
-    @JsonProperty("private")
```

Comment:
This is no longer needed, but removing the class is more involved, since it is a public class. :sob:

Reply:
Does removing this field affect backwards compatibility? I am not really sure how the class is used.

Reply:
It is actually present in the parent class.  I'm not sure what `GHRepository defines _private but doesn't get deserialized reliably with Jackson" means.  It is identical code.

Reply:
I'm not kidding.  This was fixed 6 years ago.  

https://github.com/github-api/github-api/blame/e426237c35910500d4484e644862326a79df36b5/src/main/java/org/kohsuke/github/GHRepository.java#L84

I guess if you're using an old enough version of github-api ... but wow.

Reply:
well I'm not really it's used except here..... not sure someone will complain if we remove the class... Remove makes sense as this class it's just a hack